flexget/plugins/api_trakt.py

```diff
@@ -211,6 +212,8 @@ def get_api_url(*endpoint):
 def upgrade_database(ver, session):
     if ver <= 2:
         raise db_schema.UpgradeImpossible
+    if ver <= 3:
+        table_add_column('trakt_movies', 'poster', Unicode, session)
```

Comment:
You didn't set the new version, `ver =4`. Check out series plugin for an example